compiler/src/iree/compiler/Dialect/Flow/Transforms/FusionOfTensorOps.cpp

```diff
@@ -115,11 +116,13 @@ static bool areFusableOps(MLIRContext *context, OpOperand *fusedOperand) {
         linalgConsumerOp.getNumLoops()) {
       return true;
     }
-    if (linalgConsumerOp.getNumReductionLoops() != 1 ||
-        !linalgConsumerOp.getMatchingIndexingMap(fusedOperand)
+    if (!linalgConsumerOp.getMatchingIndexingMap(fusedOperand)
              .isPermutation()) {
       return false;
     }
+    if (linalg::isaContractionOpInterface(linalgConsumerOp)) {
+      return false;
+    }
```

Comment:
also block on `isaConvolutionOpInterface` here

Reply:
One of these should be `isaConvolutionOpInterface`?

Reply:
Oh, what. Thanks for catching that